project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject vdom "0.2.2-SNAPSHOT"
+(defproject com.phronemophobic/vdom "0.2.2"
```

Comment:
Looks like this needs to be changed back, but maybe you could bump the patch version number.

Reply:
sounds good. 

> The above virtual-node? function is confusing to me, since I'd expect js/VDOM.isVirtualNode to always return true or false. Is that not the case?

You're right about this being unexpected. It's probably worth updating the comment I have in `virtual-node?` to point out that `js/VDOM.isVirtualNode` doesn't return just true/false, but potentially other js falsey values.

Reply:
I'll update the pull request tomorrow.

Reply:
Ok! Updated!

Let me know if you see any other improvements. Thanks for the library!